### PR TITLE
more policy fun

### PR DESCRIPTION
**Pull request by msuhovec**
adding more policy text and extended rodmapto policies page

### CODE REVIEW

**Comment by dbrower** [comment]
Let me take a look at this.  I'll move each policy into its own page.

File: app/views/static_pages/faqs.html.erb [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
   <h2><%= link_to 'Governing policies', policies_path %></h2>
   <ul>
-    <li><%= link_to 'CurateND Content in Scope Policy', policies_path(anchor: 'content') %></li>
-    <li><%= link_to 'General Support Policy for Depositors', policies_path(anchor: 'support') %></li>
-    <li><%= link_to 'CurateND Mission', policies_path(anchor: 'mission') %></li>
+    <li><%= link_to 'CurateND Content in Scope Policy', named_policy_path('content') %></li>
+    <li><%= link_to 'General Support Policy for Depositors', named_policy_path('support') %></li>
+    <li><%= link_to 'CurateND Mission', named_policy_path('mission') %></li>
```
Comment (danhorst) [inline app/views/static_pages/faqs.html.erb]: The order of the policies on in this listing should be the same as the policies landing page. Mission should be first.

**Comment by msuhovec** [comment]
Is it OK to send this on to staging? Do you merge it? Tag it? Don wasn't sure what the official process was.

**Comment by danhorst** [comment]
👍

**Comment by cdeluca495** [comment]
Here are some edits that should be made on the policies pages.
On the mission page:
https://curatepprd.library.nd.edu/policies/mission
Accessible is misspelled. Needs another s
Accessibility is misspelled. Needs another s

For collections by...
iMaking, needs to be changed to Making

On the content in scope page:
in the line beginning with  In addition, CurateND welcomes scholarly materials curatyed and...

curated is misspelled.

**Comment by dbrower** [comment]
I think some of these misspellings have been fixed by #253, but maybe not all of them.

**Pull request merged**